ui/webui/resources/cr_elements/cr_toggle/cr_toggle.ts

```diff
@@ -32,6 +32,7 @@ export class CrToggleElement extends CrLitElement {
       checked: {
         type: Boolean,
         reflect: true,
+        notify: true,
```

Comment:
This was the root cause of the issue - it works a bit differently to the old polymer bindings

Reply:
this is notifying the parent basically? per your comment below